finesapp/src/test/kotlin/ua/anastasiia/finesapp/GetFineByCarPlateNatsControllerTest.kt

```diff
@@ -28,17 +30,18 @@ class GetFineByCarPlateNatsControllerTest {
     @Test
     fun `should return success fine by a specific car plate`() {
         // GIVEN
-        val savedFine = fineRepository.saveFine(getFineToSave()).block()
+        val savedFine = fineRepository.saveFine(getFineToSave().toDomainFine()).block()
         val expectedResponse = GetFineByCarPlateResponse
             .newBuilder()
-            .apply { successBuilder.setFine(savedFine!!.toProto()) }
+            .apply { successBuilder.setFine(savedFine!!.toMongoFine().toProto()) }
```

Comment:
looks wierd. Maybe, we should have mapper: `domain` -> `proto`

Reply:
okay, I'm planning to add it on DDD